bubblewrap.c

```diff
@@ -482,9 +482,16 @@ acquire_privs (void)
         die_with_error ("Unable to set fsuid");
 
       /* setfsuid can't properly report errors, check that it worked (as per manpage) */
-      new_fsuid = setfsuid (-1);
-      if (new_fsuid != real_uid)
-        die ("Unable to set fsuid (was %d)", (int)new_fsuid);
+      new_fsid = setfsuid (-1);
+      if (new_fsid != real_uid)
+        die ("Unable to set fsuid (was %d)", (int)new_fsid);
+
+      if (setfsgid (real_gid) < 0)
+        die_with_error ("Unable to set fsgid");
```

Comment:
I am going to drop this patch as we discussed on IRC.

For the record, it was an error on my side as I was using "chmod +s" instead of "chmod u+s" for testing locally the bwrap executable